package.json

```diff
@@ -113,6 +115,7 @@
     "react-dom": "16.13.1",
     "react-helmet": "6.1.0",
     "react-syntax-highlighter": "13.5.3",
+    "react-window": "1.8.5",
```

Comment:
Looks like the dependency is missing for codesandbox.

Reply:
Added to `retrieveCodesandboxParameters`